UI/window-youtube-actions.cpp

```diff
@@ -125,7 +125,8 @@ OBSYoutubeActions::OBSYoutubeActions(QWidget *parent, Auth *auth,
 				const QImage newImage = imgReader.read();
 				ui->thumbnailPreview->setPixmap(
 					QPixmap::fromImage(newImage).scaled(
-						160, 90, Qt::KeepAspectRatio));
+						160, 90, Qt::KeepAspectRatio,
+						Qt::SmoothTransformation));
```

Comment:
There's a second instance of this further down, is the flag also needed there? I'm unfamiliar with this dialog.

Reply:
Ah yeah that's from loading presets, that should have the filtering as well. Updated it now.